src/mysql.erl

```diff
@@ -422,7 +422,9 @@ transaction(Conn, Fun, Args, Retries) when is_list(Args),
                 exit  -> Reason
             end,
             {aborted, Aborted}
-    end.
```

Comment:
How about exceptions raised in the `catch` clause? I think `gen_server:call(Conn, end_transaction)` is better called in an `after` clause, i.e. `try ... catch ... after gen_server:call(Conn, end_transaction) end`.